node/valcheck/aggregator.go

```diff
@@ -0,0 +1,27 @@
+package valcheck
+
```

Comment:
@nivBlox let's remove it from this PR and move it to when we have the aggregator PR

Reply:
i sugget to leave it for the next pr to save us to write the same code
@lior-blox